ThunkLibs/include/common/Guest.h

```diff
@@ -91,7 +99,12 @@ template<auto Thunk, typename Result, typename... Args>
 inline Result CallHostFunction(Args... args) {
 #ifndef _M_ARM_64
     uintptr_t host_addr;
+#if __SIZEOF_POINTER__ == 8
     asm volatile("mov %%r11, %0" : "=r" (host_addr));
+#else
+    // XXX: This function is currently unsupported for 32-bit and needs some more work.
+    assert(false && "This is currently unsupported");
```

Comment:
Can we ifdef out all unsupported functions instead?

Reply:
We could comment out the uses. It's just libvulkan and libGL which are already not being compiled for 32-bit.
I don't mind whichever path we take here. This was mostly to make sure we don't forget about it later.

Reply:
Actually, just removed this commit since it is unnecessary, since the libraries using it aren't compiled and will result in a compile error when enabled which is more useful.